<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="inputWindowLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Окно ввода</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="outputWindowLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Окно вывода</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="senderComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="receiverComboBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputWindowLine">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="senderSelectLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выбор порта передачи</string>
    </property>
   </widget>
   <widget class="QLabel" name="receiverSelectLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выбор порта приема</string>
    </property>
   </widget>
   <widget class="QLabel" name="controlWindowLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>630</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Окно управления</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="controlWindowFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>650</width>
      <height>90</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="inputWindowFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>320</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="outputWindowFrame">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>320</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="statusWindowLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>630</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Окно состояния</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="statusWindowFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>650</width>
      <height>120</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputWindowLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="statusWindowLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>630</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <zorder>inputWindowLabel</zorder>
   <zorder>outputWindowLabel</zorder>
   <zorder>senderSelectLabel</zorder>
   <zorder>receiverSelectLabel</zorder>
   <zorder>controlWindowLabel</zorder>
   <zorder>controlWindowFrame</zorder>
   <zorder>inputWindowFrame</zorder>
   <zorder>outputWindowFrame</zorder>
   <zorder>statusWindowLabel</zorder>
   <zorder>statusWindowFrame</zorder>
   <zorder>senderComboBox</zorder>
   <zorder>receiverComboBox</zorder>
   <zorder>outputWindowLine</zorder>
   <zorder>inputWindowLine</zorder>
   <zorder>statusWindowLine</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
